development/publishing_repositories.md

```diff
@@ -0,0 +1,30 @@
+## Publishing Repositories to the public organization space
+
+DRUD operates with the goal of working in publicly accessible and open-source spaces as much as possible. Projects should be given strong consideration for publishing to the open space, using the following guidelines and preparation steps to determine the best path forward for a project.
+
+### Publishing Guidelines
+
+When determining if a repository should be published to the DRUD public organization, the following questions should be considered:
+
+- Does the project provide value to others?
+- Would a person outside drud be able to understand the purpose and benefit of the project?
+- If someone didn't know about drud and discovered us from this project, would the project present us and our work in a positive light?
+- Will the project be actively maintained? If not, what level of maintenance/support do we intend to provide?
+
+If the answer to any of these questions is no, we should strongly consider making the repo private, or migrate the repo to another space if appropriate. If you're not certain if a project meets these guidelines, or if it meets some but not all of the guidelines, a ticket should be created in drud/general to discuss if the project should be made public.
+
+If a project doesn't meet the guidelines for being public in the DRUD organization, you should still consider if there is value in surfacing the work via other mediums. For example, while specific purpose-built functionality or internal documentation may not provide direct value to others offered as a code repository, it may still provide (potentially more) value in the form of a blog post or case study.
+
```

Comment:
To address @cyberswat's question on what happens when a repo that was made public no longer meets the guidelines, I suggest adding the following.

"Occasionally, a repository opened to the public may no longer meet the aforementioned guidelines. In these situations, the preference will be to keep the repository open while providing additional information on the main project page indicating any change in support, maintenance, etc. If a repository needs to be pulled back private, the team will use its best judgment in how to do so without disrupting users that may be using and/or dependent upon it.